docs/user-manual/modules/ROOT/pages/camel-jbang.adoc

```diff
@@ -1809,32 +1809,32 @@ NOTE: These are options from the export command, so you can see mor details and
 
 == Configuration
 
-Camel JBang can be used to store and use user configuration so that cli options won't have to be specified each time, for example, in order to run a different camel version, instead of executing
+Camel JBang `config` command is used to store and use the user configuration. This eliminates the need to specify CLI options each time. For example, to run a different camel version, instead of executing
 
 [source,bash]
 ----
 camel run * --camel-version=3.18.4
 ----
 
-the option camel-version can be added to the user configuration like
+the camel-version can be added to the user configuration such as:
 
 [source,bash]
 ----
 camel config set camel-version=3.18.4
 ----
 
-and then, the run command will pick the user configuration
+Now, the run command picks the user configuration:
 
 [source,bash]
 ----
 camel run *
 ----
 
-The user configuration file will be stored in ~/.camel-jbang-user.properties
+The user configuration file is stored in ~/.camel-jbang-user.properties
```

Comment:
it may be nice to put single quote around the file name so it stand out a bit better